### PR TITLE
Add ability to disable 4 named policies

### PR DESCRIPTION
**Pull request by thomas-burke**

### CODE REVIEW

File: stable/dynamic-gateway-service/templates/statefulset.yaml [hunk 1]
```diff
@@ -289,6 +289,28 @@ spec:
...
+              value: "true"
+{{- else }}
+              value: "false"
+{{- end }}
+            - name: "GWD_POLICY_DISABLE_XSLT"
+{{- if eq .Values.datapower.policyDisableXSLT "true" }}
+              value: "true"
+{{- else }}
+              value: "false"
+{{- end }}  - name: "GWD_POLICY_DISABLE_GATEWAYSCRIPT"
```
Comment (dmnewber) [inline stable/dynamic-gateway-service/templates/statefulset.yaml]: If this doesn't result in a renderer error, it's still a poor formatting practice. `-name` should be on its own line.

File: stable/dynamic-gateway-service/templates/statefulset.yaml [hunk 2]
```diff
@@ -289,6 +289,28 @@ spec:
...
+{{- if eq .Values.datapower.policyDisableGatewayScript "true" }}
+              value: "true"
+{{- else }}
+              value: "false"
+{{- end }}  - name: "GWD_POLICY_DISABLE_RATELIMIT"
```
Comment (dmnewber) [inline stable/dynamic-gateway-service/templates/statefulset.yaml]: Same as above.

**Review by dmnewber — DISMISSED** [review]

File: stable/dynamic-gateway-service/templates/statefulset.yaml (hunk 1 above)
Comment (thomas-burke) [reply stable/dynamic-gateway-service/templates/statefulset.yaml]: good catch, fixed

File: stable/dynamic-gateway-service/templates/statefulset.yaml (hunk 2 above)
Comment (thomas-burke) [reply stable/dynamic-gateway-service/templates/statefulset.yaml]: good catch, fixed

**Review by johnbellessa — DISMISSED** [review]
Please add a version bump in `stable/dynamic-gateway-service/Chart.yaml` to v1.0.33

**Comment by thomas-burke** [comment]
@johnbellessa @dmnewber I think I've addressed the comments, please review again.

**Review by johnbellessa — APPROVED** [review]
LGTM

**Review by dmnewber — APPROVED** [review]
LGTM

**Pull request merged**